### ISSUE TITLE
Documentation to install lastest v1 not working

### ISSUE THREAD

**Issue by iamhsa** [MetricMike/asdf-awscli#5]
**Describe the bug**
The install of awscli v1 as describe in [README.md](https://github.com/MetricMike/asdf-awscli/blob/main/README.md), do not work and try to install v2

**Steps to reproduce**

Try to install the latest awscli v1 as describe in [README.md]
```bash
$ asdf install awscli latest 1 # 1.19.33
awscli 2.1.31 is already installed
```
**Expected behavior**

Installation of the latest version of awscli v1

**Additional context**
Test only on linux.
Install specific version is ok :
```bash
$ asdf install awscli 1.19.33
* Downloading awscli release 1.19.33...
~/.asdf/installs/awscli/1.19.33 ~/.asdf/plugins/awscli
Collecting pip
  [...]
awscli 1.19.33 installation was successful!
```

**Comment by iamhsa** [comment]
Oops ! After looking at the [build.yml](https://github.com/MetricMike/asdf-awscli/blob/main/.github/workflows/build.yml) file I noticed that the syntax should be:
``` sh
$ asdf install awscli latest:1
```
Ultimately it's just a documentation typo in README.

Sorry I could have done it before !

**Comment by awwaiid** [comment]
I opened #12 with this change!

**Comment by MetricMike** [comment]
Closed in #12
